temporary_files_cleaner/temporary_files_cleaner.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo apt-get autoremove -y
+sudo apt-get autoclean
+sudo apt-get clean
+sudo journalctl --vacuum-time=3d
```

Comment:
1. What does this command do in the background?
2. What type of files and from where does it delete?
3. Does it deal with user-level files or `system` files?
4. What does the switch `--vacuum-time=3d` do and why the value passed is `3d` only?

Reply:
Basically those are system log files.
There are two types in which these logs can be deleted. One is by time and other by size. Here, 3d means 3 days. That is only the logs of the past three days are retained and the rest are removed.

Reply:
I accidentally deleted the repository.